doc/start/intro.rst

```diff
@@ -49,10 +49,10 @@ required when running Ceph File System clients.
   Ceph Storage Cluster.
 
 Ceph stores data as objects within logical storage pools. Using the
-:term:`CRUSH` algorithm, Ceph calculates which placement group should
-contain the object, and further calculates which Ceph OSD Daemon
-should store the placement group.  The CRUSH algorithm enables the
-Ceph Storage Cluster to scale, rebalance, and recover dynamically.
+:term:`CRUSH` algorithm, Ceph calculates which placement group should contain
+the object, and calculates which Ceph OSD Daemon should store the placement
+group. The CRUSH algorithm enables the Ceph Storage Cluster to scale,
+rebalance, and recover dynamically.
```

Comment:
From the "don't use a word twice in a sentence" perspective, I might suggest something like

":term:`CRUSH` algorithm, Ceph calculates which placement group (PG) should contain
the object and which OSD should store the PG. The CRUSH algorithm enables the Ceph Storage Cluster to scale, rebalance, and recover dynamically."

The `D` in `OSD` stands for `Daemon`, so `OSD Daemon` is redundant, like DAT Tape.  Which itself is a misnomer since DAT was for audio and DDS for data, but I digress..

Reply:
Let's make these changes in a subsequent PR. I agree with your notes, but this is scope creep.

Reply:
https://github.com/ceph/ceph/pull/46633 supersedes this PR. This PR is therefore closed.